style-manual-australian-government.csl

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Style Manual - Australian Government (author-date)</title>
+    <id>http://www.zotero.org/styles/style-manual-australian-government</id>
+    <link href="http://www.zotero.org/styles/style-manual-australian-government" rel="self"/>
+    <link href="http://www.australia.gov.au/publications/style-manual" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/harvard1" rel="template"/>
+    <author>
+      <name>jstedman</name>
+      <email>jessica.stedman@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <summary>Snooks style manual Australia Harvard author-date style</summary>
+    <updated>2014-09-10T06:18:55+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+         <terms>
+	<term name="edition" form="short">edn</term>
+</terms>  
+</locale>
+ <macro name="editor"> 
+    <names variable="editor">
+      <name and="symbol"  initialize-with=""/>
+      <label form="short" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with=""/>
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <names variable="editor" />
+        <text macro="title"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="symbol"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="title"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="URL">
+	  <group>
+          <text value="viewed"/>
+          <date variable="accessed">
+            <date-part name="day" prefix=" "/>
+            <date-part name="month" prefix=" "/>
+            <date-part name="year" prefix=" "/>
+          </date>
+        </group>
+	<text variable="URL" prefix=", &lt;" suffix="&gt;"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="bill book graphic legal_case legislation motion_picture report song thesis webpage" match="any">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else>
+        <text variable="title" text-case="title" quotes="true"/>
```

Comment:
the text-case="title" here, are you sure about that? Seems very odd that only article titles would be title-cased?